versions/run.ver

```diff
@@ -48,4 +48,6 @@ export zlib_ver=1.2.11
 export sp_ver=2.3.3
 export ip_ver=3.3.3
 
+export cmdaccel_ver=1.1
```

Comment:
What is this new module?

Reply:
See Diane's comment https://github.com/NOAA-EMC/global-workflow/pull/3144#issuecomment-2533190930